src/QsCompiler/BondSchemas/CompilerObjectTranslator.cs

```diff
@@ -201,7 +202,7 @@ bondQsBindingKind switch
                 fullName: bondQsCallable.FullName.ToCompilerObject(),
                 attributes: bondQsCallable.Attributes.Select(a => a.ToCompilerObject()).ToImmutableArray(),
                 modifiers: bondQsCallable.Modifiers.ToCompilerObject(),
-                sourceFile: bondQsCallable.SourceFile,
+                source: new Source(bondQsCallable.SourceFile, QsNullable<string>.Null),
```

Comment:
I think it would be good to encapsulate the creation of a `Source` object into a `ToCompilerObject()` method. I know it is only one line of code but it would be consistent with the pattern used in this file, and it would be less error prone in case the `Source` type changes or what we pass to the constructor changes.